docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/snowflake-logs-source.md

```diff
@@ -40,10 +40,10 @@ Once you have all the required values, set up the source configuration to collec
 
 When you create a Snowflake Logs source, you add it to a Hosted Collector. Before creating the Source, identify the Hosted Collector you want to use or create a new Hosted Collector. For instructions, see [Configure a Hosted Collector and Source](/docs/send-data/hosted-collectors/configure-hosted-collector).
 
-To configure a Trend Micro source:
+To configure a Snowflake source:
 1. [**Classic UI**](/docs/get-started/sumo-logic-ui-classic). In the main Sumo Logic menu, select **Manage Data > Collection > Collection**. <br/>[**New UI**](/docs/get-started/sumo-logic-ui). In the Sumo Logic top menu select **Configuration**, and then under **Data Collection** select **Collection**. You can also click the **Go To...** menu at the top of the screen and select **Collection**.
 1. On the Collection page, click **Add Source** next to a Hosted Collector.
-1. Search for and select **Trend Micro**.
+1. Search for and select **Salesforce**.
```

Comment:
@johnjbutler thanks for your edits! Should this be `Snowflake`, not `Salesforce`?